src/validation/aero_model/modal_aero_bcscavcoef_get.cpp

```diff
@@ -0,0 +1,68 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <mam4xx/mam4.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <skywalker.hpp>
+#include <validation.hpp>
+
+using namespace skywalker;
+using namespace mam4;
+
+void modal_aero_bcscavcoef_get(Ensemble *ensemble) {
+  ensemble->process([=](const Input &input, Output &output) {
+    const Real zero = 0;
+
+    const int ncol = int(input.get_array("ncol")[0]);
+
+    auto scavimptblvol_vector = input.get_array("scavimptblvol");
+    auto scavimptblnum_vector = input.get_array("scavimptblnum");
+
+    Real scavimptblvol[aero_model::nimptblgrow_total][AeroConfig::num_modes()] =
```

Comment:
You might not need the `zero` here--I think C++ will zero-initialize a data structure with just the curly braces. Best to verify, though.